circleci/images/pgupgradetester/files/sbin/install-testdeps

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
I am not sure, all of our images have similar scripts.

If we put each command separately to the dockerfile, rebuilds could be faster. Each RUN command will create an intermediate layer but we can at least have some parts of the script separated so that even if we change one line all the script does not get executed again.

Which one do you think is better?

Reply:
If most of our images are built like this I think we can keep it like this.